app/helpers/deadlines_helper.rb

```diff
@@ -0,0 +1,21 @@
+module DeadlinesHelper
+  def long_format_datetime datetime
+    datetime.strftime "%a, %b %d %Y %H:%M:%S"
+  end
```

Comment:
i18n có hỗ trợ format date đấy ông dùng cái đấy cho tiện

Reply:
à ok thank you